calculus/integration-partial-fractions.md

```diff
@@ -1 +1,117 @@
 # Partial fraction decomposition
+## Shortcut: Heaviside Cover-up Method
+
+$$ 
+\frac{3x+11}{(x-3)(x+2)} = \frac{A_1}{x-3} + \frac{A_2}{x+2} 
+$$
```

Comment:
```suggestion
\begin{equation}
\frac{3x+11}{(x-3)(x+2)} = \frac{A_1}{x-3} + \frac{A_2}{x+2} 
\end{equation}
```
Wrap all equations in `\begin{equation}\end{equation}` (for single equations) and `\begin{align}\end{align}` (for multiple equations). This will automatically number the equations so you can be easily refer back to them.